### DISCUSSION TITLE
Old discussion forum

### DISCUSSION THREAD

**Discussion by paolo2504** [GrandOrgue/grandorgue#940] [General]
I don't see anymore the discussion forum on Sourceforge. Please, where has it been moved ?

**Comment by rousseldenis** [comment]
> I don't see anymore the discussion forum on Sourceforge. Please, where has it been moved ?

It has been moved here.

Old discussions too.

**Reply by paolo2504** [reply]
Thanks Denis but i guess i'm missing something... I was looking for some old posts of mine so i filtered author:paolo2504.
Filter only returns my posts on github, not the old ones.
Now i'm trying to navigate all posts page by page but it tooks a lot. It seems that also searching items in the body does'nt work for old posts...
I see that without filter there are 30 pages , the older post dated 5 aug 2021.

**Reply by rousseldenis** [reply]
Could you try "by Paolo" with double quotes in filter ?

**Reply by larspalo** [reply]
All old posts are here. I get multiple hits by just searching for: by Paolo, but your username was different on Sourceforge than here.

As the old posts all were migrated in one go they all seem to be posted around the same time (by me), but that's why all the posts start with a statement who originally posted the topic and when.

**Comment by paolo2504** [comment]
Lars, Roussel: many thanks. Now I see old posts. Actually I used not nickname … I used real name on source forge.
